File: platform/College-1/Litcoder/Week 3/Code/lab11.py
def print_common():
    freq_dict = {}

    # Loop to read input 3 times
    for _ in range(3):
        in_arr = input().split(' ')
        for num in in_arr:
            num = int(num)
            freq_dict[num] = freq_dict.get(num, 0) + 1

    found = False

    # Iterate through the dictionary to find common elements
    for key, value in freq_dict.items():
        if value == 3:
            found = True
            print(f'{key} ', end='')

    if not found:
        print('No Elements', end='')

# Call the function
print_common()

File: platform/College-1/Litcoder/Week 3/Code/lab11.java
import java.util.Scanner;
import java.util.Map;
import java.util.HashMap;

public class Main {

    private void printCommon() {
        Scanner in = new Scanner(System.in);
        HashMap<String, Integer> freq = new HashMap<String, Integer>();

        // Loop to read input 3 times
        for (int iter = 0; iter < 3; iter++) {
            String[] arr = in.nextLine().split(" ");
            for (String num : arr) {
                freq.put(num, freq.getOrDefault(num, 0) + 1);
            }
        }

        boolean found = false;

        // Iterate through the HashMap to find common elements
        for (Map.Entry<String, Integer> entry : freq.entrySet()) {
            if (entry.getValue() == 3) {
                found = true;
                System.out.printf("%s ", entry.getKey());
            }
        }

        if (!found) {
            System.out.print("No Elements");
        }

        in.close();
    }

    public static void main(String[] args) {
        Main ob = new Main();
        ob.printCommon();
    }
}
